website/public/js/controllers/inventoryCtrl.js

```diff
@@ -234,6 +241,72 @@ habitrpg.controller("InventoryCtrl",
       return filteredArray;
     };
 
+    $scope.groupEquipmentBy = function(group, costume) {
+
+     var gear = {};
+
+     if (group === "stat") {
+      $scope.gear.forEach(function(item, index, array) {
+       var index = "";
```

Comment:
I can't help but think there's a way to meta program this so that we don't need to have all these if blocks.